plugins/modules/inventory_intent.py

```diff
@@ -961,10 +1306,20 @@ def get_diff_merged(self, config):
         device_type = self.config[0].get("type", "NETWORK_DEVICE")
         device_resynced = self.config[0].get("device_resync", False)
         device_updated = self.config[0].get("device_updated", False)
+        device_reboot = self.config[0].get("reboot_device", False)
         self.result['log'] = []
 
+        if device_reboot:
+            self.reboot_access_points()
+            return self
```

Comment:
I don't have a way to perform more than one work items at a time. 
For example, if  reboot devices and also export device details how can I do it?

Reply:
add the compatibility to execute multiple items in the module.